[qt5-base] build failure

**Host Environment**
- OS: Windows 10
- Compiler: MSVC++ 2019

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install qt5-base:x64-windows`

**Failure logs** 
It's not so much the building but it fails to download some of its dependencies, a broken mirror perhaps?:

```
-- Downloading https://repo.msys2.org/msys/x86_64/pkg-config-0.29.2-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/make-4.3-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libintl-0.19.8.1-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://repo.msys2.org/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirror.yandex.ru/mirrors/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirrors.ustc.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.bit.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirror.bit.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirror.selfnet.de/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirror.selfnet.de/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 22;"HTTP response code said error"
-- Downloading https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz...
-- Downloading https://mirrors.sjtug.sjtu.edu.cn/msys2/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz... Failed. Status: 28;"Timeout was reached"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:181 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues
```

**Additional context**
n/a

--- a/scripts/cmake/vcpkg_acquire_msys.cmake
+++ b/scripts/cmake/vcpkg_acquire_msys.cmake
@@ -289,8 +289,8 @@ function(vcpkg_acquire_msys PATH_TO_ROOT_OUT)
     DEPS libiconv
   )
   msys_package(
-    URL "https://repo.msys2.org/msys/x86_64/libiconv-1.16-1-x86_64.pkg.tar.xz"
-    SHA512 6f9b778d449410273a50cdd1af737cdcb8890a5536d78211477eed7382340253c7aadfb04977f1038ae4f4cef5a641f1acfda26fd06323d0b196a3e6da7fd425
+    URL "https://repo.msys2.org/msys/x86_64/libiconv-1.16-2-x86_64.pkg.tar.zst"
+    SHA512 3ab569eca9887ef85e7dd5dbca3143d8a60f7103f370a7ecc979a58a56b0c8dcf1f54ac3df4495bc306bd44bf36ee285aaebbb221c4eebfc912cf47d347d45fc
     DEPS gcc-libs
   )
   msys_package(

